internal/baggagecopy/baggagecopy.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
does anyone know what the requirements are if we copy code from apache 2.0 repos? Do we need to include the apache 2.0 license like this [blog](https://fossa.com/blog/open-source-licenses-101-apache-license-2-0/) mentions?